roles/wildfly_install/tasks/prereqs.yml

```diff
@@ -1,11 +1,29 @@
 ---
-- name: Install required packages
+- name: "Check that required packages list has been provided."
+  ansible.builtin.assert:
+    that:
+      - wildfly_prereqs_packages is defined
+    quiet: true
+    fail_msg: "Required packages list was not defined"
+
+- name: "Prepare packages list"
+  ansible.builtin.set_fact:
+    wildfly_packages_list: "{{ wildfly_prereqs_packages }}"
+  when:
+    - not wildfly_install is defined
+    - not wildfly_install.java.package_name is defined
+
+- name: "Add JDK package {{ wildfly_install.java.package_name }} to packages list"
+  ansible.builtin.set_fact:
+    wildfly_packages_list: "{{ wildfly_prereqs_packages }} + {{ wildfly_install.java.package_name }}"
```

Comment:
I think this should be "{{ wildfly_prereqs_packages  + [ wildfly_install.java.package_name ] }}" to make it a list?